test/parse/test_text.rb

```diff
@@ -21,5 +21,21 @@ def test_after_root
         DETAIL
       end
     end
+
+    def test_whitespace_characters_after_root
+      xml = <<~XML
+      <a>b</a> 
+      XML
+
+      parser = REXML::Parsers::BaseParser.new(xml)
+
+      events = {}
+      while parser.has_next?
+        event = parser.pull
+        events[event[0]] = event[1]
+      end
+
+      assert_equal("b", events[:text])
```

Comment:
Can we check all `:text` event values instead of the last `:text` event value?

Reply:
OK
I see.